### ISSUE TITLE
Support compound order when sorting manually

### ISSUE THREAD

**Issue by dur41d** [directus/app#2264]
I have a blog site and I created an article collection that I want to be sorted by `id desc` so that I can see the latest article on top. (I have more than 10,000 articles). (BTW, currently there is no way to sort table view by id so I did it in the database directly `directus_collection_presets:view_query  "sort": "-id"`). 
Also, I want to control the order of the articles manually so I created a sort field. However when I enable the manual sorting the table view puts the first article (the oldest) on top which is not what I want. This is because the values for the sort field are null and when sorting all the null fields that first in the table becomes on top. 

Basically this is the order that I would like to achieve `ORDER BY sort, id desc` but there is no way i can achieve that currently. I tried putting "sort": "sort, -id" but it didn't work). 

I had previously created a feature request https://github.com/directus/app/issues/1262 regarding the manual sort that can be used to solve this issue.

**Comment by benhaynes** [comment]
I just answered your other question to address how to sort columns in the App. As for compound API sorting, you can do that in this way:

https://docs.directus.io/api/reference.html#sorting

The App doesn't support it (as far as I know), but you could submit a Feature Request for that. If something isn't working based on the Docs, you can open a full Bug Report and we'll look into it.

**Comment by rijkvanzanten** [comment]
To achieve better clarity/visibility, we are now tracking feature requests within the [Feature Request](https://github.com/directus/app/projects/45) project board.

**This issue being closed does not mean it's not being considered.**
